apko-snapshot/README.md

```diff
@@ -22,6 +22,17 @@ The resulting image is signed with `cosign` keyless signing.
 ```yaml
 steps:
 - uses: distroless/actions/apko-snapshot@main
+  id: apko-snapshot
   with:
     base-tag: ghcr.io/distroless/foo
+
+# Pass the digest output to a trivy scan.
+- name: Run Trivy vulnerability scanner
+  uses: aquasecurity/trivy-action@master
+  with:
+    image-ref: ${{ steps.apko-snapshot.outputs.digest }}
+    format: 'table'
```

Comment:
right now this is a lie because it can't detect we are alpine:
```
2022-04-12T22:08:36.668Z	INFO	Need to update DB
2022-04-12T22:08:36.668Z	INFO	Downloading DB...
2022-04-12T22:08:38.713Z	INFO	Number of language-specific files: 0
```

vs.
```
2022-04-12T21:52:13.995Z	INFO	Need to update DB
2022-04-12T21:52:13.995Z	INFO	Downloading DB...
2022-04-12T21:52:[17](https://github.com/mattmoor/iptables/runs/5998130746?check_suite_focus=true#step:6:17).452Z	INFO	Detected OS: alpine
[20](https://github.com/mattmoor/iptables/runs/5998130746?check_suite_focus=true#step:6:20)22-04-12T21:52:17.452Z	INFO	Detecting Alpine vulnerabilities...
2022-04-12T21:52:17.457Z	INFO	Number of language-specific files: 0
```